distribution/conf/application.properties

```diff
@@ -37,8 +37,8 @@ server.port=8848
 
 ### Connect URL of DB:
 # db.url.0=jdbc:mysql://127.0.0.1:3306/nacos?characterEncoding=utf8&connectTimeout=1000&socketTimeout=3000&autoReconnect=true&useUnicode=true&useSSL=false&serverTimezone=UTC
-# db.user=nacos
-# db.password=nacos
+# db.user=root
```

Comment:
do not change the default value

Reply:
I think it should be combined with (console-src-main-resources-application.properties)The default value of is the same, but nacos:nacos Is the default value, I will change the default value to nacos:nacos